spec/models/manageiq/providers/azure/cloud_manager/event_catcher_spec.rb

```diff
@@ -3,20 +3,19 @@
     let(:unsupported_reason) { "Timeline events not supported for this region" }
 
     before do
-      @ems = FactoryBot.create(:ems_azure)
-      allow(@ems).to receive(:authentication_status_ok?).and_return(true)
-      allow(described_class).to receive(:all_ems_in_zone).and_return([@ems])
```

Comment:
my goal is to remove this stub and instead use the `authentication_status_ok?` from the database.